installation/cmd/run.sh

```diff
@@ -11,9 +11,12 @@ if [ `uname -s` = "Darwin" ]; then
     VM_DRIVER="hyperkit"
 fi
 
+source $CURRENT_DIR/../scripts/utils.sh
```

Comment:
Please extract "$CURRENT_DIR/../scripts" to a separate var

Reply:
done